ovs/extensions/healthcheck/helpers/cache.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+import json
+from ovs.extensions.storage.exceptions import KeyNotFoundException
+from ovs.extensions.storage.persistentfactory import PersistentFactory
+
+class CacheHelper(object):
+
+    client = PersistentFactory.get_client()
+    prefix = 'ovs-health-check'
+
+    @staticmethod
+    def set(info):
+        """
+        Store the information to the config management
+        :param info:
+        :return:
+        """
+
+        if isinstance(info, type(dict)):
+            info = CacheHelper._to_json(info)
+        CacheHelper.client.set(CacheHelper.prefix, info)
+
+    @staticmethod
+    def get(key=prefix):
+        try:
+            return CacheHelper._from_json(CacheHelper.client.get(key))
+        except KeyNotFoundException:
+            return None
+
+    @staticmethod
+    def _to_json(value):
```

Comment:
to_json method but you return the plain value if you receive a type error. This method will (in most cases by invalid) if a person is developing. Here you are permitted to raise a `RuntimeError` or reraising the `TypeError`. We want to know if a give json is invalid.